src/renderer/components/layout/sidebar.tsx

```diff
@@ -72,9 +72,15 @@ export class Sidebar extends React.Component<Props> {
     });
   }
 
+  @computed
```

Comment:
There is nothing to `compute` :) It's better to use when deriving new state from `observable`-s.